examples/music_mp3.rs

```diff
@@ -1,13 +1,14 @@
-extern crate rodio;
-
-use std::io::BufReader;
+use std::{fs::File, io::BufReader};
 
 fn main() {
-    let device = rodio::default_output_device().unwrap();
-    let sink = rodio::Sink::new(&device);
+	let device = rodio::default_output_device().unwrap();
```

Comment:
Rodio uses spaces for indentation.

Reply:
I planed to run `cargo fmt` at the here of this PR but right now, I got an error because of my version of Rust.
```
Warning: Unknown configuration option `fn_args_density`
Warning: Unknown configuration option `fn_brace_style`
Warning: Unknown configuration option `fn_call_style`
Warning: Unknown configuration option `fn_empty_single_line`
Warning: Unknown configuration option `generics_indent`
Warning: Unknown configuration option `impl_empty_single_line`
Warning: Unknown configuration option `reorder_imported_names`
Warning: Unknown configuration option `reorder_imports_in_group`
Warning: Unknown configuration option `where_density`
Warning: Unknown configuration option `where_style`
Warning: Unknown configuration option `wrap_match_arms`
Warning: Unknown configuration option `write_mode`
Error: Decoding config file failed:
unknown variant `Visual`, expected one of `Compressed`, `Tall`, `Vertical` for key `fn_args_layout`
Please check your config file.
```

I will fix it later.

Reply:
I planed to run `cargo fmt` at the end of this PR but right now, I got an error because of my version of Rust.
```
Warning: Unknown configuration option `fn_args_density`
Warning: Unknown configuration option `fn_brace_style`
Warning: Unknown configuration option `fn_call_style`
Warning: Unknown configuration option `fn_empty_single_line`
Warning: Unknown configuration option `generics_indent`
Warning: Unknown configuration option `impl_empty_single_line`
Warning: Unknown configuration option `reorder_imported_names`
Warning: Unknown configuration option `reorder_imports_in_group`
Warning: Unknown configuration option `where_density`
Warning: Unknown configuration option `where_style`
Warning: Unknown configuration option `wrap_match_arms`
Warning: Unknown configuration option `write_mode`
Error: Decoding config file failed:
unknown variant `Visual`, expected one of `Compressed`, `Tall`, `Vertical` for key `fn_args_layout`
Please check your config file.
```

I will fix it later.